llvm/lib/Target/RISCV/RISCVProcessors.td

```diff
@@ -242,4 +242,5 @@ def VENTANA_VEYRON_V1 : RISCVProcessorModel<"veyron-v1",
                                              FeatureStdExtZicbom,
                                              FeatureStdExtZicbop,
                                              FeatureStdExtZicboz,
-                                             FeatureVendorXVentanaCondOps]>;
+                                             FeatureVendorXVentanaCondOps,
+                                             TuneVeyronFusions]>;
```

Comment:
It looks like you're adding this to the feature list, not the tune list.  That's probably not what you meant.

Reply:
done in https://github.com/llvm/llvm-project/pull/70414